datasets/turk/turk.py

```diff
@@ -0,0 +1,124 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""TURKCorpus: a dataset for sentence simplification evaluation"""
+
+from __future__ import absolute_import, division, print_function
+
+import datasets
+
+
+_CITATION = """\
+ @article{Xu-EtAl:2016:TACL,
+ author = {Wei Xu and Courtney Napoles and Ellie Pavlick and Quanze Chen and Chris Callison-Burch},
+ title = {Optimizing Statistical Machine Translation for Text Simplification},
+ journal = {Transactions of the Association for Computational Linguistics},
+ volume = {4},
+ year = {2016},
+ url = {https://cocoxu.github.io/publications/tacl2016-smt-simplification.pdf},
+ pages = {401--415}
+ }
+}
+"""
+
+_DESCRIPTION = """\
+TURKCorpus is a dataset for evaluating sentence simplification systems that focus on lexical paraphrasing,
+as described in "Optimizing Statistical Machine Translation for Text Simplification". The corpus is composed of 2000 validation and 359 test original sentences that were each simplified 8 times by different annotators.
+"""
+
+_HOMEPAGE = "https://github.com/cocoxu/simplification"
+
+_LICENSE = "GNU General Public License v3.0"
+
+_URL_LIST = [
+    (
+        "test.8turkers.tok.norm",
+        "https://raw.githubusercontent.com/cocoxu/simplification/master/data/turkcorpus/test.8turkers.tok.norm",
+    ),
+    (
+        "tune.8turkers.tok.norm",
+        "https://raw.githubusercontent.com/cocoxu/simplification/master/data/turkcorpus/tune.8turkers.tok.norm",
+    ),
+]
+_URL_LIST += [
+    (
+        f"{spl}.8turkers.tok.turk.{i}",
+        f"https://raw.githubusercontent.com/cocoxu/simplification/master/data/turkcorpus/{spl}.8turkers.tok.turk.{i}",
+    )
+    for spl in ["tune", "test"]
+    for i in range(8)
+]
+
+_URLs = dict(_URL_LIST)
+
+
+class TurkCorpus(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class Turk(datasets.GeneratorBasedBuilder):
```
This must be the camel case version of the dataset script name `turk`

You'll need to regenerate the dataset_infos.json file after this change